packages/clay-css/src/scss/mixins/_input-groups.scss

```diff
@@ -299,6 +299,22 @@
 
 		@if ($enabled) {
 			@include clay-css($map);
+			@include clay-css(map-get($map, after));
```

Comment:
@SelenaAungst Thanks for the update, the lines 302 to 317 should be:
```scss
&::before {
	@include clay-css(map-get($map, before));
}

&::after {
	@include clay-css(map-get($map, after));
}
```

What this does is gets the `before` and `after` keys from the map `$input-group-item`. We passed the map contained in `$input-group-item: (focus: ())` right below on line 320. This mixin will print out the CSS as:

```css
.input-group-item.focus::after {
  ...
}
```
since that is all we are passing in from the variables in `_forms.scss`.

If we passed:

```scss
$input-group-item: (
    before: (
        color: red,
    ),
);
```

it would output:

```css
.input-group-item::before {
    color: red;
}
```

Reply:
Hi @pat270,
Is there anything else I can do for this?